test/functional/p2p_filter.py

```diff
@@ -93,6 +94,23 @@ def test_size_limits(self, filter_node):
 
         filter_node.send_and_ping(msg_filterclear())
 
+    def test_msg_mempool(self):
+        self.log.info("Check that a node with bloom filters enabled services p2p mempool messages")
+        self.nodes[0].disconnect_p2ps()
```

Comment:
Same commit:

I have a slight preference that tests clean up after themselves, and not after other tests. So this should be moved to the end of the subtest that created the peer connection.